src/middlewared/middlewared/plugins/tunables.py

```diff
@@ -22,6 +22,13 @@ class Config:
         register=True
     ))
     async def do_create(self, data):
+        """
+        Create a Tunable.
+
+        `var` represents name of the sysctl/loader/rc variable.
+
+        `type` should be either of LOADER / RC / SYSCTL these options.
```

Comment:
This is already known from the enum of the schema type. If we are not going to make any further explanation about what each of these mean better not to even mention it.